module/platform/OpenCR/HardwareIO/data/config/HardwareIO.yaml

```diff
@@ -94,3 +94,9 @@ servos:
     direction: 1
     offset: 0.0
     simulated: false
+
+servo:
+  temp_tol: 50.0
```

Comment:
In a config file, there are no prizes for cutting off letters. Just say temperature_alarm_level or something